bumpversion/vcs.py

```diff
@@ -126,9 +126,13 @@ def add_path(cls, path):
 
     @classmethod
     def tag(cls, sign, name, message):
+        """
```

Comment:
The PEP8 recommendation is to start the docstring with a short, descriptive sentence (with leading capital and trailing dot).

The next line, if any, should be empty followed by paragraphs.

So first line could be like "Create a tag of the new_version in VCS."

Reply:
Thanks!

Reply:
Still needs an empty line after the first one. Sorry ! :-)

Reply:
No, I am sorry!
Thanks!